src/main/java/com/almostreliable/merequester/wireless/WRMenuHost.java

```diff
@@ -0,0 +1,19 @@
+package com.almostreliable.merequester.wireless;
+
+import appeng.api.storage.ITerminalHost;
+import appeng.menu.ISubMenu;
+import appeng.menu.locator.ItemMenuHostLocator;
+
+import de.mari_023.ae2wtlib.api.terminal.ItemWT;
+import de.mari_023.ae2wtlib.api.terminal.WTMenuHost;
+
+import net.minecraft.world.entity.player.Player;
+
+import java.util.function.BiConsumer;
+
+public class WRMenuHost extends WTMenuHost implements ITerminalHost {
```

Comment:
I don't think this really matters, but since the non-wireless terminal uses implements `RequesterTerminalHost`, it might make sense to use it here too:
```suggestion
public class WRMenuHost extends WTMenuHost implements ITerminalHost {
public class WRMenuHost extends WTMenuHost implements ITerminalHost, RequesterTerminalHost {
```

Reply:
oops, I made a copy paste error here, the old line is now duplicated in addition to the new line.

Reply:
Fixed, also added a few imports to satisfy the compiler.